py3status/modules/zypper_updates.py

```diff
@@ -0,0 +1,70 @@
+"""
+Display number of pending updates for OpenSUSE Linux.
+
+Display the system updates counted by zypper lu.
+
+Configuration parameters:
+    cache_timeout: How often we refresh this module in seconds
+        (default 600)
+    format: Display format to use
+        (default 'zypper: [\?color=update {update}]')
+
+Format placeholders:
+    {updates} number of pending zypper updates
+
+Color options:
+    color_bad: out to date system
+    color_degraded: Upgrade available
+    color_good: No upgrades needed
+
+@author Ioannis Bonatakis <ybonatakis@suse.com>
+@license BSD
+
+SAMPLE OUTPUT
+{'full_text': 'zypper: 137', 'color': '#FFFF00', 'cached_until': 1616323657.0}
+
+"""
+
+import subprocess
+import re
+
+
+class Py3status:
+    """
+    """
+
+    # available configuration parameters
+    cache_timeout = 600
+    format = "zypper: [\?color=update {update}]"
+
```

Comment:
Missing `thresholds` config.
```suggestion
thresholds = [(0, "good"), (50, "degraded"), (100, "bad")]
```

Reply:
sorry for that. i dont know how i left it out. fixed